nocom/src/Account.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './InputBox.css';
+
+function InputBox({ className, onClick }) {
```

Comment:
if you want to implement class name just use a prop insert, you also don't need to pass in onClick, just make a function for it

Reply:
Like I said I am working on it
I only wanted to leave a branch on the repo for you to see since you asked

I will let you know when it is ready for review